### PR TITLE
Added jenkins checks file

### PR DESCRIPTION
**Pull request by adamgajzlerowicz**

### CODE REVIEW

File: .jenkins/checks.Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+                        }
+                    }
+                }
+            }
+        }
+    }
+    post {
+        unsuccessful {
+            script {
+                if (env.DAILY) {
```
Comment (reakaleek) [inline .jenkins/checks.Jenkinsfile]: I think you can remove the whole `post` closure. I doubt this will ever run in a daily job

**Review by reakaleek — CHANGES_REQUESTED** [review]

File: .jenkins/checks.Jenkinsfile (hunk 1 above)
Comment (adamgajzlerowicz) [reply .jenkins/checks.Jenkinsfile]: https://github.com/pmedianetwork/eslint-config/pull/79/commits/9ed621e32ee0d73bdd4ec3e5c19da4615435c894

**Comment by reakaleek** [comment]
jenkins test this please

**Review by reakaleek — CHANGES_REQUESTED** [review]
I think you have to remove the `--watch` option in the `test` script
otherwise the job will run indefinitely

File: .jenkins/checks.Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,47 @@
+pipeline {
+    libraries { lib 'adverity-shared-library@0.4.1'}
```
Comment (TkDodo) [inline .jenkins/checks.Jenkinsfile]: @reakaleek is this version okay or is there a newer version by now that we should use?

File: package.json [hunk 3]
```diff
@@ -5,7 +5,7 @@
   "author": "pmedianetwork",
   "main": "index.js",
   "scripts": {
-    "test": "jest --watch",
+    "test": "jest",
```
Comment (TkDodo) [inline package.json:8]: @adamgajzlerowicz lets add the :watch script separtly please:

`"test:watch": "npm run test -- --watch"`

**Review by TkDodo — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (adamgajzlerowicz) [reply package.json:8]: Sure 
https://github.com/pmedianetwork/eslint-config/pull/79/commits/762cc3f5052d69256d57a7711e31c37c21aad591

File: .jenkins/checks.Jenkinsfile (hunk 2 above)
Comment (reakaleek) [reply .jenkins/checks.Jenkinsfile]: The latest version 2.6.. but none of the new features are used here.. up to you

Comment (reakaleek) [reply .jenkins/checks.Jenkinsfile]: The latest version 2.6.. but none of the new features are used here.. up to you

Comment (adamgajzlerowicz) [reply .jenkins/checks.Jenkinsfile]: https://github.com/pmedianetwork/eslint-config/pull/79/commits/309753f99fe3db318e1aee1081079a1159161994

**Review by TkDodo — APPROVED** [review]

**Review by reakaleek — APPROVED** [review]

**Pull request merged**